Sources/App/WebView/IncomingURLHandler.swift

```diff
@@ -150,6 +162,34 @@ class IncomingURLHandler {
         }
     }
 
+    private func confirmAction(title: String, message: String, handler: @escaping () -> Void, cancelHandler: (() -> Void)? = nil) {
```

Comment:
`-> Promise<Void>` may make the call sites a little easier, e.g.

```swift
confirmAction(…).done {
  // do the thing
}.catch {
  // do an error
}
```

and `.cauterize()` when we don't care about the error, or in the places right now…

```swift
firstly {
  confirmAction(…)
}.done {
  // rest of the chains
}
```